tests/plugins/test_koji.py

```diff
@@ -93,20 +93,67 @@ def prepare():
     return tasker, workflow
 
 
-def test_koji_plugin():
-    tasker, workflow = prepare()
-    runner = PreBuildPluginsRunner(tasker, workflow, [{
-        'name': KojiPlugin.key,
-        'args': {
-            "target": KOJI_TARGET,
-            "hub": "",
-            "root": ROOT,
+class TestKoji(object):
+    @pytest.mark.parametrize(('root',
+                              'koji_ssl_certs',
+                              'expected_string',
+                              'expected_file'), [
+        # Plain http repo
+        ('http://example.com',
+         False,
+         None,
+         None),
+
+        # https with koji_ssl_certs
+        # ('https://example.com',
+        #  True,
+        #  'sslcacert=',
+        #  '/etc/yum.repos.d/example.com.cert'),
+
+        # https with no cert available
+        ('https://nosuchwebsiteforsure.com',
+         False,
+         'sslverify=0',
+         None),
+
+        # https with cert available
+        # ('https://example.com',
+        #  False,
+        #  'sslcacert=/etc/yum.repos.d/example.com.cert',
+        #  '/etc/yum.repos.d/example.com.cert'),
+
+    ])
+    def test_koji_plugin_http(self, tmpdir, root, koji_ssl_certs,
```

Comment:
my fault, but this should probably be renamed back to 'test_koji_plugin' since there's nothing especially 'http' about it.